fit_resonator/Sdata.py

```diff
@@ -404,7 +403,7 @@ def PlotFit(x,
             extract_factor=None,
             title="Fit",
             manual_params=None,
-            dfac: int = 1,
+            dfac: int = 10,
```

Comment:
I think this should be left as 1, downsampling the plots can be misleading for users (we had a problem today with this feature).

Reply:
Oh it's trying to revert a lot of your changes it seems, I must've done something wrong with the version control I'll update the pull request

Reply:
I think it's correct now, yes